frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -904,16 +924,24 @@ export const funnelLogic = kea<funnelLogicType>({
         ],
 
         isPropertyExcluded: [
-            () => [selectors.excludedPropertyNames],
-            (excludedPropertyNames) => (propertyName: string) =>
-                excludedPropertyNames.find((name) => name === propertyName) !== undefined,
+            () => [selectors.propertyNames],
```

Comment:
Ah, I think this is where the confusion for me about propertyNames starts from 😅 . `isPropertyExcluded` is actually `isPropertyExcludedFromProject` (worth renaming imo), so it can't use `propertyNames` as its data. Needs to be `excludedPropertyNames` ?

Reply:
Yes that clarifies things a little, will update